runtime/src/accounts_db.rs

```diff
@@ -8078,6 +8075,23 @@ impl AccountsDb {
                 )
             });
         });
+    }
+
+    /// lookup each pubkey in 'purged_slot_pubkeys' and unref it in the accounts index
+    /// populate 'purged_stored_account_slots' by grouping 'purged_slot_pubkeys' by pubkey
+    // pubkeys_removed_from_accounts_index - These keys have already been removed from the accounts index
+    //    and should not be unref'd. If they exist in the accounts index, they are NEW.
```

Comment:
are these 2 lines intentionally not doc comments?

Reply:
fixed all of them